guillotina/contrib/swagger/services.py

```diff
@@ -33,7 +33,7 @@ def get_data(self, data):
     def load_swagger_info(self, api_def, path, method, tags, service_def):
         path = path.rstrip("/")
         if path not in api_def:
-            api_def[path or "/"] = {}
+            api_def[path.replace(":path", "") or "/"] = {}
```

Comment:
I'm not sure if this is intentional for some reason (I'm not familiar with that part of guillotina) but looks weird to me checking the path is not in `api_def` and then using `path.replace(":path", "")` as key. Is this ok?

Reply:
Does it make sense to change this to `path = path.rstrip("/").replace(":path", "")`?

Reply:
the real path definition unfortunately is not valid openapi :(, i am not aware if there are better ways to handle this, but producing non standard openapi could be really dangerous.

And looks like your change makes sense, i am also non familiar with openapi generation so i wanted to change as little as possible, :P

Reply:
> the real path definition unfortunately is not valid openapi :(, i am not aware if there are better ways to handle this, but producing non standard openapi could be really dangerous.

Yeah, I'm ok replacing the non-standard part

> And looks like your change makes sense, i am also non familiar with openapi generation so i wanted to change as little as possible, :P

Me neither :)